torch_np/_normalizations.py

```diff
@@ -60,6 +61,19 @@ def normalize_axis_like(arg, name=None):
 
 
 def normalize_ndarray(arg, name=None):
+    # check the arg is an ndarray, extract its tensor attribute
+    if arg is None:
+        return arg
+
+    from ._ndarray import ndarray
+
+    if not isinstance(arg, ndarray):
+        raise TypeError("'out' must be an array")
```

Comment:
We should improve the error message using the name. Just patch this into the next PR.

Reply:
gh-109